frontend/src/pages/AuctionView.tsx

```diff
@@ -72,7 +72,10 @@ export function AuctionView() {
   // The minimum bid for the auction will be saved as minNextBid.
   const currHighestBid =
     bidHistory.length > 0 ? findHighestBid(bidHistory).amount : 0;
-  const minNextBid = Math.max(currHighestBid, auctionInfo.minimumBid);
+  const minNextBid =
+    bidHistory.length > 0
+      ? Math.max(auctionInfo.minimumBid, currHighestBid + 1)
+      : auctionInfo.minimumBid;
```

Comment:
I don't see why these extra checks are necessary. I understand the need for `+ 1` but not any of the rest. Was there some edge case you discovered that I'm not seeing?


```suggestion
  const minNextBid = Math.max(currHighestBid + 1, auctionInfo.minimumBid);
```